packages/generator-emakinacee-react/generators/app/templates/shared/dot/.eslintrc

```diff
@@ -1,7 +1,10 @@
 {
     "extends": [
         "emakinacee-base",
-        "emakinacee-react"
+        "emakinacee-react",
+        // Must be last to override rules. https://github.com/prettier/eslint-config-prettier#installation
+        "prettier",
+        "prettier/react"
```

Comment:
Aligning our rules would be a better way, yes. I tried it out with a minimal rule-set (intenitally low amount of rules): https://github.com/sladiri/browser-boiler/blob/master/.eslintrc